Readme.md

```diff
@@ -56,6 +53,36 @@ make develop
 
 When switching key pairs on a channel, the new certficiate must be built into the bundle, which must then be signed with the old key. For this purpose, the `--override-cert` option of the deploy script is needed to provide RAUC with a certificate matching the new key.
 
+## Installation on VirtualBox
+
+1. Build the installer with:
+
+```sh
+nix-build \
+  --arg buildInstaller true \
+  --arg buildBundle false \
+  --arg buildLive false \
+  --arg buildDisk false
+```
```

Comment:
Maybe a reference to the build arguments suffices for [self-help](https://en.wiktionary.org/wiki/give_a_man_a_fish_and_you_feed_him_for_a_day;_teach_a_man_to_fish_and_you_feed_him_for_a_lifetime)? https://github.com/dividat/playos/tree/develop#quick-start

Optionally adding a sub-heading to the arguments section to make it linkable might help.